### PR TITLE
DCOS-57973: adds support for vertical tabs

### PR DESCRIPTION
**Pull request by mperrotti**
## Screenshot
Screen Shot 2019-10-25 at 3 32 05 PM

### CODE REVIEW

**Comment by GeorgiSTodorov** [comment]
If I make the browser window small, I don't get a scrollbar or anything like that. Is this intentional?
Screenshot from 2019-10-28 10-55-26

**Comment by mperrotti** [comment]
@GeorgiSTodorov - Yes. The tab content just fills the remaining space

**Comment by GeorgiSTodorov** [comment]
Does that mean that we need some other element to make the tab content scrollable?

**Comment by mperrotti** [comment]
We should be switching to horizontal tabs on smaller viewports. Maybe I should make the `direction` prop responsive. e.g.: `direction={{default: "horiz", medium: "vert"}}`

**Comment by mperrotti** [comment]
Hold merge until I add the responsive support

File: packages/tabs/style.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+              css`
+                ${atMediaUp[breakpoint](css`
+                  ${getHorizOrVertStyle(direction[breakpoint])};
+                `)};
+              `
+            );
+
+            return acc;
+          },
+          Array()
```
Comment (DanielMSchmidt) [inline packages/tabs/style.ts]: Just curious, why not `[]`?

Comment (mperrotti) [reply packages/tabs/style.ts]: Typescript doesn't like `[]` for some reason ¯\_(ツ)_/¯

**Pull request merged**